source/_components/lightwave.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: "Lightwave"
+description: "Instructions on how to integrate Lightwave devices with Home Assistant."
+date: 2019-02-09 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lightwave.png
+ha_category:
+  - Hub
+  - Light
+  - Switch
+ha_release: 0.87
+ha_iot_class: "Assumed State"
+redirect_from:
+  - /components/light.lightwave/
+  - /components/switch.lightwave/
+---
+
+The `lightwave` component links Home Assistant with your Lightwave WiFi link for controlling Lightwave lights and switches.
+
+This component uses the official API published by Lightwave on their website [https://api.lightwaverf.com/](https://api.lightwaverf.com/).
+To add your Lightwave devices into your Home Assistant installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+lightwave:
+  host: 192.168.1.2
+  lights:
+    R1D3:
+      name: Wall lights
+    R1D4:
+      name: Ceiling lights
+  switches:
+    R1D2:
+      name: Tree socket
+    R2D1:
+      name: Radio socket
+    R2D2:
+      name: Light socket
+    R2D3:
+      name: Phone socket
+    R2D4:
+      name: Torch socket
+    registration:
+      name: Registration
+```
+### `host`
+Where `192.168.1.2` is the ip address of your Lightwave hub.
```

Comment:
is this as example?

Reply:
@klaasnicolaas I am going to try again as I want to get this right. I think the steps I need are:
1. fork
2. create a branch
3. do the work
4. commit
5. Raise the PR

Also done on this now.